src/csvcubed/utils/qb/validation/observations.py

```diff
@@ -142,32 +187,50 @@ def _attribute_represents_observation_status(attribute: QbAttribute) -> bool:
 def _validate_observation_value(
     observation_value: QbColumn[QbObservationValue],
     multi_unit_columns: List[QbColumn[QbMultiUnits]],
+    num_obs_val_columns: int,
 ) -> List[ValidationError]:
     errors: List[ValidationError] = []
+    is_single_obs_val_data_set = False
```

Comment:
`is_single_obs_val_data_set = num_obs_val_columns == 1` would simplify these multiple lines down to one.